x-pack/plugins/ml/common/constants/data_frame_analytics.ts

```diff
@@ -10,3 +10,12 @@ export const ANALYSIS_CONFIG_TYPE = {
   CLASSIFICATION: 'classification',
 } as const;
 export const DEFAULT_RESULTS_FIELD = 'ml';
+
+export const JOB_MAP_NODE_TYPES = {
+  ANALYTICS: 'analytics',
+  TRANSFORM: 'transform',
+  INDEX_PATTERN: 'index-pattern',
```

Comment:
Is the map actually looking for Kibana index patterns, or Elasticsearch indexes? Wondering as I deleted the `outliers-clientip-03` index pattern here, but the node still shows up on the map. Maybe the legend should be labelled as `index` rather than `index-pattern` ?

![image](https://user-images.githubusercontent.com/7405507/98363186-7644d280-2026-11eb-8748-eff7bed90123.png)

Reply:
Oh good catch - it's the index - I've updated it to reflect that 👍